### DISCUSSION TITLE
Getting User IP Address in Nuxt 3 SSR Application

### DISCUSSION THREAD

**Discussion by piyushladhar** [nuxt/nuxt#27755] [Questions]
Our use case involves showing pricing based on geolocation. So a customer from India would see pricing in INR and other countries customer would see in USD. For that, we want to get the IP address or country of the visitor.
We were using HTML,CSS and PHP till now. With PHP we could get the client IP easily using REMOTE_ADDR and then use an API to find the country of that IP address.

I am not finding any solution for Nuxt 3. We are using Nuxt 3 Typescript. And it's a SSR app, we thought to use an API to get IP address but I think that would return the server IP address instead.

Can anyone help me in this regards?

**Comment by manpreet-compro** [comment]
@piyushladhar  In the server APIs, I believe you could get the IP from h3event by using `getRequestIP` function
https://h3.unjs.io/utils/request#getrequestipevent

Let me know if this helps
